wagtailio/utils/blocks.py

```diff
@@ -222,12 +226,12 @@ class StoryBlock(StreamBlock):
         icon="doc-full-inverse",
         template="patterns/components/streamfields/document/document.html",
     )
-    imagecaption = ImageAndCaptionBlock(label="Image caption")  # to be removed
-    textimage = TextAndImageBlock(icon="image")  # uses text_and_media_block.html
-    colourtext = BackgroundColourTextBlock(icon="pilcrow")  # to be removed
-    calltoaction = CallToActionBlock(icon="pilcrow")  # to be removed
-    tripleimage = TripleImageBlock(icon="image")  # to be removed
-    stats = ListBlock(StatBlock(icon="code"))  # to be removed
+    imagecaption = ImageAndCaptionBlock(label="Image caption", group= "DO NOT USE")  # to be removed
+    textimage = TextAndImageBlock(icon="image", group= "DO NOT USE")  # uses text_and_media_block.html
+    colourtext = BackgroundColourTextBlock(icon="pilcrow", group= "DO NOT USE")  # to be removed
+    calltoaction = CallToActionBlock(icon="pilcrow", group= "DO NOT USE")  # to be removed
+    tripleimage = TripleImageBlock(icon="image", group= "DO NOT USE")  # to be removed
+    stats = ListBlock(StatBlock(icon="code"), group= "DO NOT USE")  # to be removed
```

Comment:
I am pretty sure I cleaned up the pages that were using these blocks, hence the `# to be removed` comments.

Will run a check tomorrow will update this

Reply:
I can confirm no pages use these anymore, so please remove with prejudice.
Don't forget to remove the actual block definitions and the corresponding templates.

Locally, I suggest running:
```
./manage.py migrate blog 0011
./manage.py migrate standardpage 0006
```
removing the migrations in this PR, then regenerating the migration once the cleanup is done